spec/controllers/reviews_controller_spec.rb

```diff
@@ -59,12 +56,12 @@
         end
       end
     end
-
-    context "with unauthenticated user" do
-      it "redirects to log in page" do
-        video = Fabricate(:video)
-        post :create, video_id: video.id, review: Fabricate.attributes_for(:review)
-        expect(response).to redirect_to log_in_path
+    
+    it_behaves_like "require_log_in" do
+      let(:action) do
+        post :create,
+        video_id: Fabricate(:video),
+        review: Fabricate.attributes_for(:review)
```

Comment:
The indentation isn't correct here. Keep hash keys together but not at the level of the method they are passed in.